PuzzleSafari/Pages/About.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@page
```

Comment:
The projects in the Puzzleday and Puzzlehunt directories are all vestigial and honestly should be removed. There's certainly no need for you to add more directories at this level.

Reply:
Toasted.

---
In reply to: [295580620](https://github.com/PuzzleServer/mainpuzzleserver/pull/491#discussion_r295580620) [](ancestors = 295580620)